coalib/results/Diff.py

```diff
@@ -244,10 +243,9 @@ def original(self):
         """
         Retrieves the original file.
         """
-        return self._file
+        return self._generate_linebreaks(self._file)
 
-    @property
-    def modified(self):
+    def _modified(self):
```

Comment:
maybe call it `_raw_modified` or so? It's hard to tell where the difference is between this one and `modified` if you don't know about this `'\n'` conversion thing^^